apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+
+### `current_token_ownerships_v2`
+
+_Has an aggregate view for summary data called `current_token_ownerships_v2_aggregate`_
+
+This table tracks who owns which NFTs. This includes both v1 and v2 tokens. Fungible tokens are not tracked as consistently.
+
+| Field                      | Type   | Description                                                                                                                              |
+| -------------------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| amount                     | bigint | The amount of the token owned. Ex. 42                                                                                                    |
+| aptos_name                 | String | This is a name tied to this account using the Aptos Name Service (ANS). Ex. "JohnDoe"                                                    |
+| collection_data_id_hash    | String | This is a hash identifying the data of the collection. Ex. "0xbd0818e81f08ccb0297a2ae80726e5f1a36ef077ad910d5c1efac79a824899f3"          |
+| collection_name            | String | The name of the collection to which the token belongs. Ex. "CryptoKitties"                                                               |
+| creator_address            | String | This is an Aptos account address that created the token. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a"        |
+| current_collection_data    | Table  | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| current_token_data         | Table  | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| last_transaction_timestamp | String | Timestamp of the last transaction involving the token. Ex. "2024-04-17T02:14:25.68771"                                                   |
+| last_transaction_version   | bigint | The version number of the last transaction involving the token. Ex. 9000000001                                                           |
+| name                       | String | This is a name tied to this token, not necessarily unique. Ex. "GoldenDragon"                                                            |
+| owner_address              | String | This is an Aptos account address that currently owns the token. Ex. "0xa815a9a09105973084bfc31530e7c8f002846787c2f0521e1e34dc144ad83b89" |
+| property_version           | bigint | The version number of the token's properties, which can change over time. Ex. 2                                                          |
+| table_type                 | String | The type of the token, usually differentiating between fungible and non-fungible. Ex. "NonFungible"                                      |
```

Comment:
This is actually type_type_v1. I think that we should remove it actually. This is the type of storage where the token lives. For token_standard v1, this is always 0x3::token::TokenStore. For v2 it's null. So it's a largely useless field.